packages/core/src/bodl/bodl-window.ts

```diff
@@ -0,0 +1,14 @@
+export interface BodlEventsCheckout { 
+    emit(eventName: string | symbol, ...args: any[]): boolean;
```

Comment:
Does this need to be `any[]` here, given the two events we have above they seem to be an object? Unless I am missing something

Reply:
No, it's basically coming from here. https://www.npmjs.com/package/events
But yeah, I'm not using it in other case than object